src/common/models/Projection.ts

```diff
@@ -88,9 +88,14 @@ export interface CaseDensityRange {
  * of 0.5% and 1.5% respectively, used to calculate case density by deaths (main
  * series and range).
  */
-const CASE_FATALITY_RATIO_LOWER = 0.005;
-const CASE_FATALITY_RATIO_UPPER = 0.015;
+// TODO: We were intending to calculate a low/high range for
+// caseDensityByDeath, based on a range of CFRs, but this doesn't work when
+// we merge with caseDensityByCases which has no range. So for now,
+// we are punting.
+// const CASE_FATALITY_RATIO_LOWER = 0.005;
+// const CASE_FATALITY_RATIO_UPPER = 0.015;
```

Comment:
I left the CFR range stuff in here but commented out in case we want to bring it back, but I can also just delete it if you prefer.

Reply:
I think we can leave it for now, we will need to clean up more things if we confirm that the uncertainty intervals (we can reuse `ChartZones` instead of having a custom chart for case density, for example)